hedera-mirror-rest/transactions.js

```diff
@@ -111,6 +112,7 @@ const createTransferLists = (rows) => {
         result: row.result,
         scheduled: row.scheduled,
         token_transfers: createTokenTransferList(row.token_transfer_list),
+        transaction_bytes: utils.encodeBase64(row.transaction_bytes),
```

Comment:
Will update this per scrum chat
```suggestion
        bytes: utils.encodeBase64(row.transaction_bytes),
```